### PR TITLE
[Storage Blob Service] Container Lease and Blob Lease

### PR DESCRIPTION
**Pull request by JinmingHu-MSFT**

### CODE REVIEW

File: sdk/storage/inc/blobs/blob_client.hpp [hunk 1]
```diff
@@ -284,6 +284,76 @@ namespace Azure { namespace Storage { namespace Blobs {
     Azure::Core::Response<UndeleteBlobInfo> Undelete(
         const UndeleteBlobOptions& options = UndeleteBlobOptions()) const;
 
+    /**
+     * @brief Acquires a lease on the blob.
+     *
+     * @param proposedLeaseId
+     * Proposed lease ID, in a GUID string format.
```
Comment (katmsft) [inline sdk/storage/inc/blobs/blob_client.hpp:291]: The format of the comment here is really weird.

Comment (JinmingHu-MSFT) [reply sdk/storage/inc/blobs/blob_client.hpp:291]: It is weird

Comment (katmsft) [reply sdk/storage/inc/blobs/blob_client.hpp:291]: Can you try to fix it?

Comment (JinmingHu-MSFT) [reply sdk/storage/inc/blobs/blob_client.hpp:291]: not a priority

**Pull request merged**